spec/all.globalBeforeEach.spec.js

```diff
@@ -0,0 +1,91 @@
+  describe('Testing - CFTO UK Site', function() {
+
+
+function log_in()
+{
+  browser.setValue('#loginEmail', 'sroy@sapient.com');
+  browser.setValue('#loginPassword', 'welcome1');
+}
+
+function isMerge()
+{
+  var el = browser.getTitle();
+  if(el == 'My Basket')
+  {
+  browser.click('button[data-analyticsfunction="analyticsCheckoutButtonClicked"]');
+  browser.setValue('.collection--store-finder__search-form__input', 'Covent Garden');
+  }
+  else {
+  browser.setValue('.collection--store-finder__search-form__input', 'Covent Garden');
+  }
+}
+
+
+it.only('Place an UK CFTO order with registered user with added item', function() {
+
+    this.timeout(0);
```

Comment:
As discussed, This is needed as some of the JS functions needs timeout to be predefined. 
Either those can be handled using promises or completely nullifying the timeout by using this function